contracts/governance/Staking/modules/StakingStakeModule.sol

```diff
@@ -182,7 +182,12 @@ contract StakingStakeModule is IFunctionsList, StakingShared, CheckpointsShared,
      * @param previousLock The old unlocking timestamp.
      * @param until The new unlocking timestamp in seconds.
      * */
-    function extendStakingDuration(uint256 previousLock, uint256 until) external whenNotPaused {
+    function extendStakingDuration(uint256 previousLock, uint256 until)
+        external
+        whenNotPaused
+        whenNotFrozen
+    {
+        previousLock = _timestampToLockDate(previousLock);
```

Comment:
This function will revert if user input 0 value for the previousLock?

Reply:
Yes, it will fail, added test to check.